sklearn/cross_validation.py

```diff
@@ -1681,9 +1682,13 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
             test_score = error_score
             if return_train_score:
                 train_score = error_score
-            warnings.warn("Classifier fit failed. The score on this train-test"
+            warnings.warn("Estimator fit failed. The score on this train-test"
                           " partition for these parameters will be set to %f. "
-                          "Details: \n%r" % (error_score, e), FitFailedWarning)
+                          "Details: \n%s" % (
+                            error_score,
+                            format_exception_only(type(e), e)[0]
+                          ), 
```

Comment:
The PR is failing on travis because of a trailing whitespace on Line 1690.

Reply:
Fixed

Reply:
Great. Try going through the CI logs to solve such problems.

Reply:
Thanks for pointing out, I'm new to this. Will keep that in mind